docs/notifications/03-showrunners-scaffold/03-Examples/08-Showrunner-Example-Time-Interval-Price-Alerts.mdx

```diff
@@ -0,0 +1,223 @@
+---
+id: docs-slider-settings-with-showrunners-example
+title: Time Interval Price Alerts
+hide_title: true
+slug: ./price-alerts-with-showrunners-example
+displayed_sidebar: pushNotificationSidebar
+sidebar_position: 7
+image: "/assets/docs/previews/docs_notifications_tutorials_push_react_component_playground_example--push_sdk_react_playground.png"
+---
+
+# Slider Notification Settings in Showrunners
+
+This example is intended to get you understand slider based notification settings with a real-world application. For the example we are going to look at a scenario where users can choose a time interval and showrunners framework will notify them as per their request. Checkout [Showrunners Docs](https://push.org/docs/notifications/showrunners-scaffold/get-started/), [Showrunners Framework](https://github.com/ethereum-push-notification-service/push-showrunners-framework/tree/main), [Channel Settings Docs](https://push.org/docs/notifications/build/create-channel-settings/) and [Channel Settings Demo](https://github.com/ethereum-push-notification-service/push-showrunners-framework/tree/main/src/sample_showrunners/bank) for better understanding!
+
+<head>
+  <title>
+    Price Alerts with Showrunners tutorial | Push Notification | Push Documentation
+  </title>
+</head>
+
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+## What we gonna build?
+
+Imagine you are a crypto trader or a general crypto enthusiast. You want to be notified every once in a while about the price movements and activities in the market. But you either lose track of time or forget about it. To solve this exact problem, we will be looking into a slider type notification settings implementation where you as a user can specify the time interval after which you would like to get notified.
+
+We will choose the [Eth Tracker](https://github.com/ethereum-push-notification-service/push-showrunners-channels/tree/wallet-tracker/Staging_channels/ethTicker) channel to demonstrate this example.
```

Comment:
Url should include title in markdown format for example [ETH Tracker](https://whatever.com "Title which makes it SEO friendly goes here")